packages/klighd-core/src/skgraph-models.ts

```diff
@@ -38,6 +38,7 @@ export const LABEL_TYPE = 'label'
  * Represents the Sprotty version of its java counterpart in KLighD.
  */
 export class SKNode extends KNode {
+    // TODO this might be wrong, interactiveLayout is enabled on the parent (I think)
```

Comment:
There's a lot of uncertainty in this comment.

Reply:
It is correct, the comment should be removed